config/betterquesting/DefaultQuests.json

```diff
@@ -306452,6 +306452,415 @@
           }
         }
       }
+    },
+    "2897:10": {
+      "questID:3": 2897,
+      "preRequisites:11": [
+        38,
+        74
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "snd_complete:8": "random.levelup",
+          "taskLogic:8": "AND",
+          "partySingleReward:1": 0,
+          "visibility:8": "NORMAL",
+          "isMain:1": 0,
+          "simultaneous:1": 0,
+          "icon:10": {
+            "id:8": "TConstruct:rapier",
+            "Count:3": 1,
+            "tag:10": {
+              "InfiTool:10": {
+                "BaseDurability:3": 249,
+                "Head:3": 14,
+                "BaseAttack:3": 5,
+                "ToolEXP:4": 0,
+                "HarvestLevel:3": 4,
+                "Attack:3": 5,
+                "RenderHead:3": 14,
+                "ModDurability:5": 0.0,
+                "Handle:3": 14,
+                "Broken:1": 0,
+                "Shoddy:5": 0.0,
+                "RenderHandle:3": 14,
+                "Accessory:3": 6,
+                "MiningSpeed:3": 650,
+                "RenderAccessory:3": 6,
+                "ToolLevel:3": 1,
+                "Unbreaking:3": 3,
+                "Damage:3": 0,
+                "BonusDurability:3": 0,
+                "TotalDurability:3": 249,
+                "Modifiers:3": 0
+              },
+              "display:10": {
+                "Name:8": "§fBronze Rapier"
+              }
+            },
+            "Damage:2": 0,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "§2§lMaking a Rapir",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Problems killing the Litch? Can\u0027t you manage to shoot back his explosive bullets? \nNo problem, just use a Rapier that ignores armor so that the Litch will soon be killed. \nThe only problem is that the Rapier does not do a lot of damage."
```

Comment:
Litch->Lich
Can\u0027t you manage->Can\u0027t manage
shoot back his explosive bullets->shoot back his projectiles
just use a Rapier that ignores armor so that the Litch will soon be killed->just use a Rapier that ignores armor and the Lich will soon be...re-dead? Un-undead?
does not do->doesn't do

Reply:
changed